lib/tasks/truncate_tables.rake

```diff
@@ -0,0 +1,13 @@
+namespace :db do
```

Comment:
what is the purpose of this rake?
Very dangerous rake to be committed in git

Reply:
Not required so removed. This was created for one time purpose to purge data from local db.